docs/rules/no-implicit-coercion.md

```diff
@@ -0,0 +1,96 @@
+# Disallow the type conversion with shorter notations. (no-implicit-coercion)
+
+On JavaScript, there are notations which are not understandable to beginners.
```

Comment:
In JavaScript, there are a lot of different ways to convert variable types. Some of them might be hard to read and understand. (or something like that).